src/components/FileUploader/styles.js

```diff
@@ -0,0 +1,86 @@
+import styled, { css } from 'styled-components';
+import FormElementWrapper from '../FormComponents/FormElementWrapper';
+import UiText from '../UiText';
+
+const styles = {
+  textColor({ invalid, valid }) {
+    if (invalid) {
+      return 'var(--f-color-element--error)';
+    }
+    if (valid) {
+      return 'var(--f-color-element--success)';
+    }
+    return 'var(--f-color-text--secondary)';
+  },
+  borderColor({ invalid, valid, hasFile }) {
+    if (hasFile) {
```

Comment:
Could we use onliners here?
IMO it would make to whole file more readable

```
borderColor({ invalid, valid, hasFile }) {
    if (hasFile) return 'transparent'
    if (invalid) return 'var(--f-color-element--error)';
    if (valid) return 'var(--f-color-element--success)';
    return 'var(--f-color-element--tertiary)';
  },
```

Reply:
I didn't think of it, looks a lot better! 👍

Reply:
💃